libmemcached/byteorder.cc:66:10: error: use of undeclared identifier 'ntohll'
libmemcached/byteorder.cc:75:10: error: use of undeclared identifier 'htonll'
--- libmemcached/byteorder.cc.orig	2014-02-09 05:52:42.000000000 -0600
+++ libmemcached/byteorder.cc	2017-12-08 11:49:13.000000000 -0600
@@ -35,6 +35,9 @@
  *
  */
 
+#include <sys/types.h>
+#include <netinet/in.h>
+#include <inttypes.h>
 #include "mem_config.h"
 #include "libmemcached/byteorder.h"
 
